manifests/u/UniversityOfWaikato/Weka/3.8.6/UniversityOfWaikato.Weka.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: UniversityOfWaikato.Weka
+PackageVersion: 3.8.6
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
`x86` validations are performed on a 32-bit OS, so the error above might indicate that this is a 64-bit program.

---

Thanks for the suggestion @stephengillie . It does install to "C:\Program Files\Weka-3-8-6"